vhost/CHANGELOG.md

```diff
@@ -1,20 +1,22 @@
 # Changelog
 ## [Unreleased]
+### Fixed
```

Comment:
Please prepare the unreleased section with all subsections, not just Fixed and Deprecated

Reply:
Thank you. I got it confused. It is now fixed.